src/body-bottom/tracking-internal.js

```diff
@@ -87,10 +87,10 @@
 	 * @returns {void}
 	 */
 	function trackPageView(context) {
-		const sessionId = M.cookie.get('tracking_session_id'),
-			pvNumber = M.cookie.get('pv_number'),
-			pvNumberGlobal = M.cookie.get('pv_number_global'),
-			cookieDomain = M.getFromShoebox('runtimeConfig.cookieDomain');
+		const sessionId = M.cookie.get('tracking_session_id');
+		const pvNumber = M.cookie.get('pv_number');
+		const pvNumberGlobal = M.cookie.get('pv_number_global');
+		const cookieDomain = M.getFromShoebox('runtimeConfig.cookieDomain');
```

Comment:
👍  if you ask me it's better this way

Reply:
It's also better if you ask our guidelines ;) We use this setting: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L358